ucsf_webedit_lite/ucsf_webedit_lite.features.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// This file is deprecated and can be removed.
+// Please remove include_once('ucsf_webedit_lite.features.inc') in ucsf_webedit_lite.module as well.
```

Comment:
@Quevin thanks for confirming. please leave this to me, i'll turn this around by EOD today.